lib/bugsnag/report.rb

```diff
@@ -23,13 +23,16 @@ class Report
     attr_accessor :api_key
     attr_accessor :app_type
     attr_accessor :app_version
+    attr_accessor :breadcrumbs
     attr_accessor :configuration
     attr_accessor :context
     attr_accessor :delivery_method
     attr_accessor :exceptions
     attr_accessor :hostname
     attr_accessor :grouping_hash
+    attr_accessor :message
```

Comment:
Similar to the comment a few lines after this, I think you can remove this and change your `#summary` method to use `exceptions[0][:message]`